work/lessonbuilder_merge_page.py

```diff
@@ -37,7 +37,7 @@ def run(SITE_ID, APP):
                 if item.attrs['type'] == '5':
```

Comment:
Please make the selection / flow logic a bit clearer in what follows here with some comments, maybe restructure into separate ifs or switch